src/vsr/grid.zig

```diff
@@ -465,10 +465,21 @@ pub fn GridType(comptime Storage: type) type {
             grid.superblock.storage.on_next_tick(.lsm, callback, next_tick);
         }
 
-        /// Returning null indicates that there are not enough free blocks to fill the reservation.
-        pub fn reserve(grid: *Grid, blocks_count: usize) ?Reservation {
+        /// Panics if there are not enough free blocks to fill the reservation.
```

Comment:
```suggestion
        /// Aborts if there are not enough free blocks to fill the reservation.
```

we are not actually using Zig's panic mechanism here